### PR TITLE
Added a guidance for installing gsplat on Windows

### PR DESCRIPTION
**Pull request by KevinXu02**

### CODE REVIEW

**Review by liruilong940607 — CHANGES_REQUESTED** [review]
This is nice! Maybe move this instruction to a standalone file, e.g., `docs/INSTALL_WIN.md`, and only put a pointer in README? 

I personally prefer this way to keep the avoid overwhelming information in the readme

**Comment by KevinXu02** [comment]
> This is nice! Maybe move this instruction to a standalone file, e.g., `docs/INSTALL_WIN.md`, and only put a pointer in README?
> 
> I personally prefer this way to avoid overwhelming information in the readme

Updated. Thanks for the advice!

File: docs/INSTALL_WIN.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+1. Install Visual Studio Build Tools. If MSVC 143 does not work, you may also need to install MSVC 142 for Visual Studio 2019. And your CUDA environment should be set up properly.
+
+2. Activate your conda environment:
+    ```bash
+    conda activate <your_conda_environment>
+    ```
+    Replace `<your_conda_environment>` with the name of your conda environment. For example:
+    ```bash
+    conda activate nerfstudio
```
Comment (liruilong940607) [inline docs/INSTALL_WIN.md:15]: Just to avoid confusion, it might be better to put `conda activate gsplat` as the example

**Review by liruilong940607 — APPROVED** [review]
LGTM!

**Pull request merged**